resources/web/wwi/webots.js

```diff
@@ -34,6 +34,8 @@ webots.User1Authentication // password or authentication for the main user (empt
 webots.User2Id             // ID of the secondary user (in case of a soccer match between two different users). 0 or unset if not used.
 webots.User2Name           // user name of the secondary user.
 webots.CustomData          // application specific data to be passed to the simulation server
+webots.showRevert          // if set to true the revert button is displayed
+webots.hideQuit            // if set to true the quit button is not displayed
```

Comment:
This doesn't look very consistent. Shouldn't we call all these variables `webots.showSomething` or `webots.hideSomething` rather than having a mixture of both?